### PR TITLE
Deep fryer fix, done by Carbonhell orignally in #1045

### PR DESCRIPTION
**Pull request by TDcoolguy300**
I fixed it up, so hopefully it will fix the shit he fixed.

### CODE REVIEW

**Comment by TDcoolguy300** [comment]
This is a fixed copy of #1045

**Comment by Carbonhello** [comment]
Yea,all my code is here,did you check if it works and compiles? If not i will in 30 minutes

File: code/modules/food&drinks/kitchen machinery/deep_fryer.dm [hunk 1]
```diff
@@ -71,7 +70,6 @@
 			F.icon = HI
 		if(!(findtext(F.name, "deep fried"))) // deep fried deep fried deep fried deep fried deep fired John Snow says, "hi", no
```
Comment (Carbonhell) [inline code/modules/food&drinks/kitchen machinery/deep_fryer.dm]: Removethis check and the statement after this,just setting the name has no effect and changing real name fucks a lot of things up,so there's no need to keep it, after this the PR is good to go

Comment (danhibiki) [inline code/modules/food&drinks/kitchen machinery/deep_fryer.dm]: Lets get these two lines patched up and it'll be good to merge.

Comment (TDcoolguy300) [inline code/modules/food&drinks/kitchen machinery/deep_fryer.dm]: Just got home, will do it now

**Comment by TDcoolguy300** [comment]
Done

**Pull request merged**